.github/dependabot.yml

```diff
@@ -1,6 +1,9 @@
 version: 2
 updates:
   - package-ecosystem: "gomod"
+    directories:
+      - "/"
+      - "/sdk/go"
```

Comment:
Do we need to add all runtime modules under SDKs to here?

Reply:
Hm, *perhaps*? When the runtime modules are run, all their dependencies that are shared with dagger itself will be auto-bumped to the right version.

So if there are other third party deps, then maybe we should - no real objection.

Reply:
Make sense. :)